src/v2/Apps/Authentication/__tests__/authenticationRoutes.jest.tsx

```diff
@@ -22,10 +23,23 @@ jest.mock("../Utils/helpers", () => ({
   setCookies: jest.fn(),
 }))
 jest.mock("v2/Utils/Hooks/useAuthValidation")
+jest.mock("v2/Utils/getENV", () => ({
+  getENV: jest.fn(),
+}))
 
 describe("authenticationRoutes", () => {
   const mockCheckForRedirect = checkForRedirect as jest.Mock
   const mockSetReferer = setReferer as jest.Mock
+  const mockGetENV = getENV as jest.Mock
+
+  mockGetENV.mockImplementation(key => {
+    switch (key) {
+      case "APP_URL":
+        return "https://artsy.net"
+      case "ALLOWED_REDIRECT_HOSTS":
+        return "off.artsy.net"
+    }
+  })
```

Comment:
unless strictly necessary being here I'd move this block down into the test where its relevant, just so future tests don't accidentally collide with another's expectation